### PR TITLE
Spice things up a little.

### PR DESCRIPTION
**Pull request by peterkagey**

### CODE REVIEW

**Comment by mzemel** [comment]
This causes my MacBook to hang. :(

**Comment by peterkagey** [comment]
Get a faster computer.

File: lib/game.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def initialize
     self.grids = [Grid.new]
     loop do
       step!
-      sleep 1
+      sleep 0.1
```
Comment (peterkagey) [inline lib/game.rb:8]: Do you think this is the change that causes your computer to hang? Or can you isolate it elsewhere?

**Comment by mzemel** [comment]
It's whatever this character `⬤` is.  Changing it back to `•` is fine.  Also, I don't believe getting rid of strings made an improvement in its memory footprint.

``` ruby
irb(main):004:0* require 'objspace'
=> true
irb(main):005:0> ObjectSpace.memsize_of("alive") == ObjectSpace.memsize_of(Cell.new)
=> true
```

**Comment by peterkagey** [comment]
I'm shocked that `⬤` is the culprit. It looked so wonderful! I'll do another commit to fix it.

Anyhow, I was after code quality, not performance.

**Comment by mzemel** [comment]
I changed the character back -- thanks for Making Open Source Great Again, Peter

**Pull request merged**